### PR TITLE
1.20 client mappings

### PR DESCRIPTION
**Pull request by bluebear94**

### CODE REVIEW

File: mappings/net/minecraft/screen/AnvilScreenHandler.mapping [hunk 1]
```diff
@@ -1,4 +1,12 @@
 CLASS net/minecraft/class_1706 net/minecraft/screen/AnvilScreenHandler
+	FIELD field_30751 MAX_NAME_LENGTH I
+	FIELD field_41894 INPUT_1_X I
+	FIELD field_41895 INPUT_2_X I
+	FIELD field_41896 OUTPUT_X I
+	FIELD field_41897 SLOT_Y I
+	FIELD field_41898 INPUT1_ID I
+	FIELD field_41899 INPUT2_ID I
```
Comment (YanisBft) [inline mappings/net/minecraft/screen/AnvilScreenHandler.mapping]:
```suggestion
	FIELD field_41898 INPUT_1_ID I
	FIELD field_41899 INPUT_2_ID I
```
just to match fields 41894 and 41895

File: mappings/net/minecraft/screen/GrindstoneScreenHandler.mapping [hunk 2]
```diff
@@ -2,6 +2,13 @@ CLASS net/minecraft/class_3803 net/minecraft/screen/GrindstoneScreenHandler
 	FIELD field_16772 input Lnet/minecraft/class_1263;
 	FIELD field_16773 result Lnet/minecraft/class_1263;
 	FIELD field_16775 context Lnet/minecraft/class_3914;
+	FIELD field_30794 INPUT1_ID I
+	FIELD field_30795 INPUT2_ID I
```
Comment (YanisBft) [inline mappings/net/minecraft/screen/GrindstoneScreenHandler.mapping]:
```suggestion
	FIELD field_30794 INPUT_1_ID I
	FIELD field_30795 INPUT_2_ID I
```

File: mappings/net/minecraft/screen/MerchantScreenHandler.mapping [hunk 3]
```diff
@@ -2,6 +2,17 @@ CLASS net/minecraft/class_1728 net/minecraft/screen/MerchantScreenHandler
 	FIELD field_18669 levelProgress I
 	FIELD field_18670 leveled Z
 	FIELD field_19358 canRefreshTrades Z
+	FIELD field_30830 INPUT1_ID I
+	FIELD field_30831 INPUT2_ID I
```
Comment (YanisBft) [inline mappings/net/minecraft/screen/MerchantScreenHandler.mapping]:
```suggestion
	FIELD field_30830 INPUT_1_ID I
	FIELD field_30831 INPUT_2_ID I
```

File: mappings/net/minecraft/screen/MerchantScreenHandler.mapping [hunk 4]
```diff
@@ -2,6 +2,17 @@ CLASS net/minecraft/class_1728 net/minecraft/screen/MerchantScreenHandler
...
+	FIELD field_30832 OUTPUT_ID I
+	FIELD field_30833 INVENTORY_START I
+	FIELD field_30834 INVENTORY_END I
+	FIELD field_30835 HOTBAR_START I
+	FIELD field_30836 HOTBAR_END I
+	FIELD field_30837 INPUT1_X I
+	FIELD field_30838 INPUT2_X I
```
Comment (YanisBft) [inline mappings/net/minecraft/screen/MerchantScreenHandler.mapping]:
```suggestion
	FIELD field_30837 INPUT_1_X I
	FIELD field_30838 INPUT_2_X I
```

**Review by YanisBft — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**